src/collectors/rmq_cycles.ts

```diff
@@ -22,7 +22,7 @@ export default class RMQCyclesConsumer {
   }
 
   public async cleanUp(): Promise<void> {
-    await this.consumer.cleanUp()
+    await this.consumer.cleanUp(true)
```

Comment:
Refer this [comment](https://github.com/shardeum/collector/pull/55#discussion_r1957068893)